client/components/UI/Preview/ExpandableText.jsx

```diff
@@ -1,52 +1,66 @@
-import React, {useRef, useState} from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import {get} from 'lodash';
 
 import {gettext} from '../../../utils';
 
 import './style.scss';
 
-export function ExpandableText({value, className, expandAt}) {
-    const valueRef = useRef();
-    const [expanded, setExpanded] = useState(false);
+export class ExpandableText extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {expanded: false};
+        this.dom = {parent: null};
+        this.toggleExpanded = this.toggleExpanded.bind(this);
+        this.setDomRef = this.setDomRef.bind(this);
+    }
 
-    const toggleExpanded = () => {
-        setExpanded(!expanded);
-        if (valueRef.current && valueRef.current.parentNode) {
-            valueRef.current.parentNode.scrollIntoView();
+    toggleExpanded(expanded) {
```

Comment:
you're not using that argument. Also, that would be an Event right?

Reply:
Good point I missed that, blame it on copy and paste!!!